index.html

```diff
@@ -227,20 +242,27 @@ <h2>Defining an RDF Dataset Hash</h2>
-     </p>
+     </p>-->
 
 
     <p>
-      The main challenge for the Working Group is to provide a standard for the <a href="#canonicalization">RDF Dataset Canonicalization function</a>. 
+      The main challenge for the Working Group was to provide a standard for the <a href="#canonicalization">RDF Dataset Canonicalization function</a>. 
     </p>
 
     <div id="noProblem" class=note>
       <p>When the hash of a file is transferred from one system to another and the file is used <em>as is</em>, there is no need for any processing other than checking the value of the hash of the file. This is true for RDF just as for any other data formats; this means that any existing hashing functions may be used on the original file. However, RDF has many serializations for datasets, notably <a href="https://www.w3.org/TR/trig/">TriG</a>, <a href="https://www.w3.org/TR/json-ld/">JSON-LD</a>, <a href="https://www.w3.org/TR/n-quads/">N-Quads</a> or, informally, <a href="https://digitalbazaar.github.io/cbor-ld-spec/">CBOR-LD</a>. The <a href="#spaceEfficient">space efficient verification</a> use case points to a need in some circumstances to transform &mdash; usually to minimize &mdash; the data that is transferred. In this scenario, a hash on the original file, such as a hash calculated on a JSON-LD file, is not appropriate, as the conversion will make it invalid. A hash of the abstract dataset, on the other hand, will still be valid.</p>
     </div>
-  </section>
+<p>
+</section>
+<section id="specDevelopment"><h2>Development of the specification</h2>
+The working group considered the algorithms proposed by Aidan Hogan [[hogan-2017]] and by Rachel Arnold and Dave Longley [[arnold-longley-2020]]. The latter was the basis for the final report on the topic published by the <a href="https://www.w3.org/community/credentials/">Credentials Community Group</a> [[CCG-RDC-FINAL]]. It was largely through that effort that the Arnold/Longley approach enjoyed significant implementation experience before the working group was chartered, backed up by a test suite. It was for this reason that, after careful consideration of the alternatives, efforts focused on formalizing the Arnold/Longley approach.</p>
+<p>A major issue that the working group had to tackle was precisely what would the serialized output of the algorithm be that could then be hashed? The current  N-Triples Recommendation [[n-triples-20140225]] includes the definition of a <a href="https://www.w3.org/TR/n-triples/#canonical-ntriples">canonical form</a>. However, the same is not true for N-Quads [[n-quads-20140225]]. This is expected to be added to a forthcoming version of the N-Quads specification but, for now, normative text is included in the RDF Dataset Canonicalization document. It is necessary to serialize the algorithm output as N-Quads, rather than N-Triples, as the input and output are RDF Datasets, which may include multiple graphs.</p>
```

Comment:
```suggestion
<p>One major issue that the working group had to tackle was to determine precisely what the serialized output of the algorithm would be, that could then be hashed. The current  N-Triples Recommendation [[n-triples-20140225]] includes the definition of a <a href="https://www.w3.org/TR/n-triples/#canonical-ntriples">canonical form</a>. However, this is not so for N-Quads [[n-quads-20140225]] — i.e., there is no canonical form of N-Quads. Such a canonical form is expected to be added to a forthcoming version of the N-Quads specification but, for now, suitable normative text is included in the RDF Dataset Canonicalization document. It is necessary to serialize the algorithm output as N-Quads, rather than N-Triples, as the input and output are RDF Datasets, which may include multiple graphs, which cannot be serialized as N-Triples.</p>
```